### PR TITLE
Revamp bot

### PR DESCRIPTION
**Pull request by AipioxTechson**
Adds:
- leaderboard
- New Displays
- Foundation for multiple game modes
- Sanitizer for quotes, used for deny listing and removing invalid quotes

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -3,28 +3,148 @@
...
+    embed=discord.Embed(title="So close!", description="You weren't able to guess the quote, that sucks!", color=0x702eb2)
+    embed.add_field(name="Original Quote", value=origQuote, inline=True)
+    return embed
+
+def DisplayCorrect(ans = None, origQuote = None, namelessQuote = None):
+    embed=discord.Embed(title="Correct!", description="Spot on, you got it right!", color=0x702eb2)
+    embed.add_field(name="Original Quote", value=origQuote, inline=True)
+    return embed
+
+def DisplayIncorrect():
```
Comment (nivykani) [inline bot.py]: to avoid clutter, maybe the bot could instead react to the message with an X emoji?

Comment (AipioxTechson) [reply bot.py]: Would we follow the same convention for the winner as well?

File: bot.py [hunk 2]
```diff
@@ -3,28 +3,148 @@
...
-global name_alias
-name_alias = {"orange": ["nivy"], "iandur": ["lance", "lander"], "laptop monkey": ["kyo", "desktop monkey"], "emily w": ["emily", "goose"],
-"blossom": ["naaz"], "onion": ["onion", "ritvik", "aipiox"], "jc": ["jerry"], "absurdism": ["krish"], "redvilder": ["nick"],"doomgooey": ["sergey", "doom"], "jonjonnho": ["jonathan"], "みお" : ["mio"],
-"luciars": ["shiba", "lucas"], "garboguy": ["liam", "garbo" ,"mailman"], "dripbot": ["bolt"], "the-call-of-the-void": ["jessica", "obomo"], "voidlord": ["chonky","chonky birb", "StarLight "], "givemewater": ["water"], "kay911kay": ["broke&homeless", "broke", "daniel"], "salazareo": ["daniel", "salazar"], 
-"riddle": ["kana"], "theraghavsharma": ["raghav", "kyo's butler"], "winghawk": ["georges","shanker"], "lukasz345": ["lukasz"], "starlight": ["chonky","chonky birb", "voidlord"],
-"1!":["wan"], "iloveubb":["hans"], 
-"undefined_kimchi": ["kimchi", "kim"], "cluelessKimchi一IlirFan": ["kimchi", "kim"], "kimchi_bot": ["kimchi", "kim"], "Chonky Rebel BOT Kimchi": ["kimchi", "kim"], "cluelessKimchi-IlirFan": ["kimchi", "kim"]
+# The different playable games
+gameStates = {
+    'guessGame': {
```
Comment (nivykani) [inline bot.py]: i think this doesn't have to be a nested dict?

File: bot.py [hunk 3]
```diff
@@ -3,28 +3,148 @@
...
+def parseGameOne(message, clientUser, ans, origQuote, namelessQuote):
+    global currentGame
+    name = None 
+    messageList = message.split(' ')
+    if len(messageList)  > 1:
+        name = ' '.join(messageList[1:]).lower()
+    #check if they try to initiate here
+    if message in ['gq','guessquote']:
+        #Trying to be sneaky
+        return discord.Embed(title='Oh no!', description='You tried to start a new game, when a current game is already in session! Use -quote to repeat the quote, or -giveup to quit the game!', color=0x702eb2)
```
Comment (nivykani) [inline bot.py]: what would you think about: 
```
A game is already in progress! Use -guess to guess or -giveup to quit the game. 
Quote: {namelessQuote}
```

File: sanitizer.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
+from old_quotes import quotes
```
Comment (nivykani) [inline sanitizer.py:1]: love this sanitizer 🙌

File: bot.py (hunk 3 above)
Comment (AipioxTechson) [reply bot.py]: Love it!

File: sanitizer.py (hunk 4 above)
Comment (AipioxTechson) [reply sanitizer.py:1]: hehe thanks

**Pull request merged**